examples/sgemm/SGEMM_main.cpp

```diff
@@ -243,7 +255,7 @@ int main(int argc, char **argv) {
     init_matrix_rand(matrixB, N, K, 10.0f);
     float *matrixC;
     matrixC = (float *)malloc(M * K * sizeof(float));
-    init_matrix_rand(matrixC, M, K, 0.0f);
```

Comment:
why is this change?

Reply:
Code cleanup. I thought that using rand with limit 0.0f is a little bit of overkill.  init_matrix(..., value) seems more logical here, although functionally it's the same thing. Additionally, the init_matrix() function was not used in the code, so we should either remove or use it.

Reply:
Ah, you are right! We definitely need to use init_matrix() here.